diktat-rules/src/test/resources/test/smoke/src/main/kotlin/Example6Expected.kt

```diff
@@ -0,0 +1,9 @@
+package org.cqfn.diktat
+
+val foo =
+        """
+            some
```

Comment:
This example is incorrect: when you moved string, you changed it. We should fix only strings that have `timIndent` or `trimMargin` at the end, because then it would be safe.

Reply:
Fixed